quickstarts/Embeddings.ipynb

```diff
@@ -35,18 +35,34 @@
     },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        output_dimensionality=10)
</pre><p> </p><p>It sounds like this change is to add this parameter but it's hidden in this code snippet comparing task types without explanation.</p><p> </p><p>Maybe add a section after this block with a short note? e.g.:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">## Truncating embeddings

The text-embedding-004 model also supports lower embedding dimensions. Specify output_dimensionality to truncate the output.


`
result1 = genai.embed_content(
&nbsp; &nbsp; model="models/text-embedding-004",
&nbsp; &nbsp; content="Hello world)


result2 = genai.embed_content(
&nbsp; &nbsp; model="models/text-embedding-004",
&nbsp; &nbsp; content="Hello world",
&nbsp; &nbsp; output_dimensionality=10)


(len(result1), len(result2))
`
</pre><p> </p><p>Can we talk about the relationship between the index and specificity? It'd be great to add a statement like "When using <code>text-embedding-004</code>, each dimension adds diminishing value so truncating may be effective in constrained environments ." - but I haven't verified if this is true.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
@markmcd I haven't verified it either but I think it would be a great add! @MarkDaoust what do you think?